nestkernel/source_table_position.h

```diff
@@ -41,14 +41,14 @@ struct SourceTablePosition
   long tid;    //!< thread index
   long syn_id; //!< synapse-type index
   long lcid;   //!< local connection index
+
   SourceTablePosition();
   SourceTablePosition( const long tid, const long syn_id, const long lcid );
   SourceTablePosition( const SourceTablePosition& rhs );
 
-  template < typename T >
-  void wrap_position( const std::vector< std::vector< BlockVector< T > > >& sources );
-
-  bool is_at_end() const;
+  void seek_to_next_valid_index( const std::vector< std::vector< BlockVector< Source > > >& sources );
```

Comment:
Please add doxygen documentation for new methods.

Reply:
ok